source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -61,14 +62,39 @@
                 }
             });
         } else {
-            //TODO: show alert popup
-            alert("HAY CALIFICACIONES INVÁLIDAS");
+            var modalContainer = $('#evaluations-generated-confirm');
```

Comment:
Well, it is not full Knockout, but, it is an step:

* https://github.com/CommonJobs/CommonJobs/blob/eval-form-develop/source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/PeriodEvaluation.cshtml#L59
* https://github.com/CommonJobs/CommonJobs/blob/eval-form-develop/source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js#L231

Maybe you can find an improvement for this code too.